CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ set(PACKAGE_NAME "RDMA")
 set(PACKAGE_VERSION "20.0")
 # When this is changed the values in these files need changing too:
 #   debian/libibverbs1.symbols
-set(IBVERBS_PABI_VERSION "19")
+set(IBVERBS_PABI_VERSION "20")
```

Comment:
Why? This is only changed when the IBVERBS_PRIVATE symbols are altered, I didn't see any of those change in this series?